python/model/material/kitchen.yaml

```
@@ -0,0 +1,34 @@
+import:
+  - ./apartment_provided.yaml
+  - ./amazon_parts.yaml
+
+assembly:
+  - name: Appliances
```

The '-' here is causing the yaml to parse this value as a list as opposed to the expected key-value pair of the fields, `name`, `description`, and `parts`. The suggested fix allows AaC to successfully validate the `model/material/my_apartment.yaml` file for me.

I'll take the action to identify helpful error messages and cases for situations like this so that issues like this are more clear to the user.
```suggestion
  name: Appliances
```